src/main.rs

```diff
@@ -152,27 +152,23 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             event: WindowEvent::Resized(new_size),
             ..
         } => {
-            size = new_size.to_physical(window.hidpi_factor());
```

Comment:
We still need to have size updated here, otherwise the new swapchain created will have an incorrect size.

Reply:
It is updated with

```rust
editor.update_size(new_size)
```

I just removed it because `new_size` is already physical. Or maybe I just don't understand something:

```
155 |             size = new_size.to_physical(window.scale_factor());
    |                             ^^^^^^^^^^^ method not found in `winit::dpi::PhysicalSize<u32>`
```

Reply:
Right, the physical conversion is no longer needed with the new winit.
Unfortunately the size kept in the editor is different than `size` here. Both will need to be updated. Below this line, we recreate a new swapchain based on `size` which needs to be set to `new_size`.
This should be:
```
size = new_size;
```

Reply:
Right